audit-triplestore-web/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="
+       http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+       http://camel.apache.org/schema/spring http://camel.apache.org/schema/spring/camel-spring.xsd">
+
+    <bean id="properties" class="org.apache.camel.component.properties.PropertiesComponent">
+        <property name="location" value="classpath:audit.properties"/>
+    </bean>
+
+    <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+      <property name="brokerURL" value="tcp://localhost:61616"/>
```

Comment:
@escowles I take that back about the `dynamic` portion of the config. That should remain there.

Should it also be `fcrepo.dynamic.jms.host`? The current fcrepo configs use `fcrepo.jms.host`. I'm inclined to keep it in line with the fcrepo convention, though I'd note that it seems inconsistent.